src/components/DpFlyout/DpFlyout.vue

```diff
@@ -2,13 +2,9 @@
   <span
     ref="target"
     v-on-click-outside="close"
-    class="o-flyout"
+    class="dp-flyout relative"
```

Comment:
Does this class (dp-flyout) exist?

Reply:
> Does this class (dp-flyout) exist?

this class (as well as the `dp-flyout-trigger` and so on) is intended to serve as a styling hook for "brute force overrides" for projects, where overriding css vars is not sufficient. It could be used in this way:

```
header > nav .dp-flyout {
  // Project specific stuff
}
```

I'd suggest to put it into all of our components, as a standard. We already discussed this with @salisdemos and while he suggested to use `data-cy` alternatively, i feel that using a real selector for this purpose is more appropriate.

Reply:
Thanks for explaining! I wonder if there's a good location to document this? Otherwise it might be confusing to find this class that is not defined anywhere

Reply:
totally agree - however i have no clue besides putting a paragraph in the welcome sections regarding component overrides.

Reply:
I think that's a good idea :+1:

Reply:
here we go be3889189ca1852d53eff83fa7ea4c80f6034463